tests/UI/pages/BO/international/locations/index.js

```diff
@@ -285,5 +313,73 @@ class Zones extends BOBasePage {
       wantedStatus ? this.bulkEnableLink : this.bulkDisableLink,
     );
   }
+
+  /**
+   * Sort table
+   * @param page
+   * @param sortBy, column to sort with
+   * @param sortDirection, asc or desc
+   * @return {Promise<void>}
+   */
+  async sortTable(page, sortBy, sortDirection) {
+    let columnSelector;
+
+    switch (sortBy) {
+      case 'id_zone':
+        columnSelector = this.sortColumnDiv(2);
+        break;
+
+      case 'name':
+        columnSelector = this.sortColumnDiv(3);
+        break;
+
+      default:
+        throw new Error(`Column ${sortBy} was not found`);
+    }
+    const sortColumnButton = `${columnSelector} i.icon-caret-${sortDirection}`;
+    await this.clickAndWaitForNavigation(page, sortColumnButton);
+  }
+
+  /* Pagination methods */
+  /**
+   * Get pagination label
+   * @param page
+   * @return {Promise<string>}
+   */
+  getPaginationLabel(page) {
+    return this.getTextContent(page, 'ul.pagination.pull-right li.active a');
```

Comment:
Can you create a var for this selector please

Reply:
Nice catch thank you :+1: done